css/default.css

```diff
@@ -631,6 +631,7 @@ hr {
 	}
 	#mainContent {
 		left: 250px; /* = #mainContent is loaded before sidebar following guidelines (more important), so shift to right for width of #side(bar)*/
+                bottom: 0px; /* ensure that mainContent does not extend below bottom of viewing pane */
```

Comment:
This entry is redundant.

The real change for mainContent needs to be made at about line 873.